docs/02-app/01-building-your-application/05-optimizing/01-caching.mdx

```diff
@@ -0,0 +1,18 @@
+
+| Mechanism                | Purpose                                                                                                                                    | What is cached?                                                        | Where is it cached?               | Duration                                                                                                           | APIs                                                                                   |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------- |
+| Static Rendering         | Reduce the rendering cost of content that doesn’t change. All routes are statically rendered by default unless they use dynamic functions. | Pre-rendered Client Components, and Rendered Server Components Payload | Server (CDN)                      | Persistent or revalidate lifetime                                                                                  | Route Segment Config, and Dynamic Functions                                            |
+| Client-side Router Cache | Improve navigation performance. Stores RSC payloads, previously fetched and soon to be fetched (prefetch)                                  | Rendered Server Component Payload                                      | Client (in-memory)                | Single Session (cleared by hard navigation or refresh), or 30s for dynamic routes, and 5 minutes for static routes | `<Link>`, `router.prefetch`, `router.refresh`, `revalidatePath`, `revalidateTag`       |
+| Next.js Data Cache       | Store data across deployments, reduce data store calls                                                                                     | Data                                                                   | Server (CDN)                      | Persistent or revalidate lifetime                                                                                  | `fetch.cache`, `fetch.revalidate`, `unstable_cache`, `revalidatePath`, `revalidateTag` |
+| Extended fetch API       | Deduplicate requests for the same data across a route render pass. Internally uses React.cache                                             | Data                                                                   | Server (in-memory)                | Per request / render pass                                                                                          | `fetch`                                                                                |
```

Comment:
It should also be noted that the `Extended fetch API` does cache data accross deployments automatically, and some API to modify it is to use the `cache` option or `next: {revalidate}` option to modify its behavior.

Reply:
Thanks for the suggestion, still iterating, we'll expand on each in the sections below.